pkg/migration/controllers/migration.go

```diff
@@ -1162,52 +1165,62 @@ func (m *MigrationController) prepareApplicationResource(
 func (m *MigrationController) prepareCRDClusterResource(
 	migration *stork_api.Migration,
 	object runtime.Unstructured,
-	suspend stork_api.SuspendOptions,
+	suspendOpts []stork_api.SuspendOptions,
+
 ) error {
 	if *migration.Spec.StartApplications {
 		return nil
 	}
+	if len(suspendOpts) == 0 {
+		return nil
+	}
 	content := object.UnstructuredContent()
-	fields := strings.Split(suspend.Path, ".")
-	var currVal string
-	if len(fields) > 1 {
-		var disableVersion interface{}
-		if suspend.Type == "bool" {
-			disableVersion = true
-		} else if suspend.Type == "int" {
-			curr, found, err := unstructured.NestedInt64(content, fields...)
-			if err != nil || !found {
-				return fmt.Errorf("unable to find suspend path, err: %v", err)
+	annotations, found, err := unstructured.NestedStringMap(content, "metadata", "annotations")
+	if err != nil {
+		return err
+	}
+	if !found {
+		annotations = make(map[string]string)
+	}
+	for _, suspend := range suspendOpts {
+		logrus.Debugf("suspend path: %v/%v", suspend.Path, suspend.Type)
```

Comment:
Do we need this log line?

Reply:
nope, since default logs level in stork is `debug` lets remove this